src/Twilio/Rest/Client.php

```diff
@@ -266,6 +313,24 @@ public function getRegion(): string {
         return $this->region;
     }
 
+    /**
+     * Retrieve the Edge
+     *
+     * @return string Current Edge
+     */
+    public function getEdge(): string {
+        return $this->edge;
+    }
+
+    /**
+     * Set Edge
+     *
+     * @param string $uri Edge to use, unsets the Edge when called with no arguments
+     */
+    public function setEdge(string $edge = null): void {
+        $this->edge = $edge;
```

Comment:
Wondering if unsetting this value (`setEdge(null)`) should cause it to revert to the env var if set?

Reply:
+1